Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:latest
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update
+RUN apt-get install -y vim
+RUN apt-get install -y zip
+RUN apt-get install -y ssh
+RUN apt-get install -y npm
+WORKDIR /app
+COPY . .
+# you can use it with:
+# docker build . -t cloudfront-auth && docker run -it cloudfront-auth
+# then, copy the zip file with:
+# docker cp <docker-id>:/app/distributions/<distribution-id> .
```

Comment:
could use more docs on the cli commands also the chosen base image requires significant install time of rpm, perhaps a node base image would be most suitable